src/function/open-referral-function/src/FamilyHubs.OpenReferral.Function/Entities/ServiceJson.cs

```diff
@@ -0,0 +1,7 @@
+namespace FamilyHubs.OpenReferral.Function.Entities;
+
+public class ServiceJson
```

Comment:
nitpick: Good candidate for a record, rather than a class :)

Reply:
Doneeee : [b829f5b](https://github.com/DFE-Digital/fh-services/pull/63/commits/b829f5bcbec1acec6d5854a3cd1de18d2031a668)